browserstack.browsers.js

```diff
@@ -1,4 +1,15 @@
 const browserstackBrowsers = {
+  // iOS
+  iphone: {
```

Comment:
@bengummer I think  `client.useIframe` might be a potential fix, because you get a new browser window. Currently trying that out.